doc/index.rst

```diff
@@ -8,42 +8,36 @@ Welcome to DeepSpeech's documentation!
 
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
+   :caption: Inroduction
```

Comment:
The long form documentation (.rst files) are very out-of-date. For example, they still talk about a bidirectional LSTM. We should either update them or remove them entirely.

Reply:
Your call. I was thinking we could update them after, for example.

Reply:
OK. I'll file an issue